src/commands/upload.js

```diff
@@ -114,9 +114,7 @@ export const handler = async (args) => {
 
   console.info(`Found ${fileList.length} file${fileList.length === 1 ? '' : 's'} ...`);
 
-  for (const fileInfo of fileList) {
-    await uploadFile(fileInfo);
-  }
+  await Promise.all(fileList.map(uploadFile));
```

Comment:
Can we chunk this? I don't think we want to do 100s of files in parallel in the worst case.

Reply:
Also, how does this affect error reporting? Are we going to spam the same error message if something is down?

Reply:
How many do we want to do at once ? `10` ?

Reply:
Sounds reasonable to me.

Reply:
I think each call will `console.error` the name of the file if it fails to upload.